input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md

```diff
@@ -0,0 +1,15 @@
+### Usage scenarios
+TBD
+
+### Profile specific implementation guidance
+- `ServiceRequest.code` represents the type of service being requested or ordered, such as a diagnostic test or a procedure. Depending on the context provided by `ServiceRequest.category`, `ServiceRequest.code` **SHOULD** be selected from a corresponding domain-specific value set:
```

Comment:
The first point can be removed, and allow for content on pathology and imaging to be 1st level points.

Reply:
Done